systemd-swap

```diff
@@ -1,613 +1,724 @@
-        UNIT_NAME=${UNIT_PATH##*/}
-        swapoff "${DEV}"
-        rm -vf "${UNIT_PATH}"
-      fi
-    done
-
-    [[ -f ${WORK_DIR}/swapfc/.lock ]] && rm -f "${WORK_DIR}/swapfc/.lock"
-
-    echo "$swap_units" | while read -r UNIT_PATH; do
-      if grep -q swapfc "${UNIT_PATH}"; then
-        DEV=$(GET_WHAT_FROM_SWAP_UNIT "${UNIT_PATH}")
-        UNIT_NAME=${UNIT_PATH##*/}
-        swapoff "${DEV}"
-        rm -vf "${UNIT_PATH}"
-        [[ -f ${DEV} ]] && rm -f "${DEV}"
-      fi
-    done
-
-    echo "$swap_units" | while read -r UNIT_PATH; do
-      if grep -q zram "${UNIT_PATH}"; then
-        DEV=$(GET_WHAT_FROM_SWAP_UNIT "${UNIT_PATH}")
-        UNIT_NAME=${UNIT_PATH##*/}
-        INFO "Zram: swapoff ${DEV}"
-        swapoff "${DEV}"
-        rm -vf "${UNIT_PATH}"
-        zramctl -r "${DEV}"
-      fi
-    done
-    [[ -v ZRAM_ALREADY_SET ]] || modprobe -r zram
-
-    if [[ -d ${WORK_DIR}/zswap ]]; then
-      INFO "Zswap: restore configuration: start"
-      for file in "${WORK_DIR}/zswap"/*; do
-        read_l "${file}" > "${ZSWAP_M_P}/${file##*/}"
-      done
-      INFO "Zswap: restore configuration: complete"
-    fi
-
-    wait
-
-    rm -f "${LOCK_STARTED}"
-  ;;
-  status)
-    declare -a swap_stats
-    get_mem_stat_multi swap_stats SwapTotal SwapFree
-    SWAP_USED=$(( swap_stats[0] - swap_stats[1] ))
-    unset swap_stats
-
-    if [[ -d /sys/module/zswap ]]; then
-      echo Zswap:
-      read -r USED_BYTES < /sys/kernel/debug/zswap/pool_total_size
-      USED_PAGES=$(( USED_BYTES / PAGE_SIZE ))
-      read -r STORED_PAGES < /sys/kernel/debug/zswap/stored_pages
-      STORED_BYTES=$(( STORED_PAGES * PAGE_SIZE ))
-      RATIO=0
-      (( STORED_PAGES > 0 )) && RATIO=$(( USED_PAGES * 100 / STORED_PAGES ))
-      {
-        grep . /sys/module/zswap/parameters/* | \
-          cut -d'/' -f6 | \
-          sed -e 's/:/ /g' -e 's/^/. /g'
-      } | column -t
-      {
-        grep . /sys/kernel/debug/zswap/* | \
-          cut -d'/' -f6 | \
-          sed -e 's/:/ /g' -e 's/^/. . /g' | column -t
-        echo . . compress_ratio ${RATIO}%
-        (( SWAP_USED > 0 )) && \
-          echo . . zswap_store/swap_store ${STORED_BYTES}/${SWAP_USED} $(( STORED_BYTES * 100 / SWAP_USED ))%
-      } | column -t
-    fi
-
-    if zramctl | grep -q '\[SWAP\]'; then
-      echo ZRam:
-      zramctl | \
-        grep -e '^NAME\|\[SWAP\]' | \
-        sed -e 's/\(MOUNTPOINT\|\[SWAP\]\)$//g' -e 's/^/.\t/g' | \
-        column -t | \
-        uniq
-    fi
-
-    if [[ -d ${WORK_DIR}/swapd ]]; then
-      echo swapD:
-      swapon --raw | \
-        grep -v 'zram\|file\|loop' | \
-        sed 's/^/.\t/g' | \
-        column -t
-    fi
-
-    if [[ -d ${WORK_DIR}/swapfc ]]; then
-      echo swapFC:
-      swapon --raw | \
-        grep -e 'NAME\|file\|loop' | \
-        sed 's/^/.\t/g' | \
-        column -t
-    fi
-  ;;
-  *)
-    help
-  ;;
-esac
-
-exit 0
-# vim: set ft=sh ts=2 sw=2 et:
+#!/bin/env python3
```

Comment:
Shouldn't this still be `#!/usr/bin/env`?

Reply:
Correct.